unit2-java/lesson14.rst

```diff
@@ -4,11 +4,17 @@ Unit 2 (Java) - Lesson 14 Learning Objectives
 Goals
 -----
 
+- Understand how to structure database tables
+- Connect database tables in one-to-many relationships
+- Write more complex SQL queries
+
 Objectives
 ----------
 
-- Utilize *phpMyAdmin* and build a database locally
-- Create a new user and database
-- Import tables using a `.sql` file
 - Explain how and why data is split into related tables
+- Utilize foreign keys
+- Create tables with an understanding of one-to-many relationships
 - Perform complex SQL queries with joins and subqueries
+- Import table data with *MySQL Workbench*
+- Write simple queries using left, right, and full joins
```

Comment:
Remove "simple", since joins fall under the "complex" category for queries.

Reply:
Done!